sphinx/source/docs/user_guide/examples/categorical_heatmap_unemployment.py

```diff
@@ -30,7 +30,7 @@
 p.rect(x="Year", y="Month", width=1, height=1, source=source,
        line_color=None, fill_color=transform('rate', mapper))
 
-color_bar = ColorBar(color_mapper=mapper, location=(0, 0),
+color_bar = ColorBar(color_mapper=mapper,
```

Comment:
just making sure I understand: the default location for colorbars is now outside the plot? I'm 👍 for that, but needs a release note

Reply:
The location is governed by `plot.add_layout()`, with `"center"` being the default. This `location=(0, 0)` was a hack (at least historically) needed to position color bars "decently" within side panels.

Reply:
what happens to existing code that adds `location` is is silently ignored? A warning might be nice but definitely don't want to error as that would break things

Reply:
Location works if color corbar is positioned on a central panel and ignored (well, assumed `[0, 0]` with `bottom_left` anchor) otherwise. Location doesn't make much sense in panels, at least in the bivariate form. However, I could implement this, especially in combination with fixed width/height.

Reply:
I don't have a feel for how many people might set `location` to something besides `(0,0)`. I expect lots of people set specifically to `(0,0)` from copy-pasting examples, so my main concern is that that case behaves reasonably without or without being specific and that seems to be what you are saying. I would suggest making a separate issue about implementing `location` for side panels rather than add more to this big PR

Reply:
I added support for `location=[x, 0]` (horizontal case) and `location=[0, y]` (vertical case) in side panels. Non-zero other component also works, but its meaning is dubious, given this is implemented using margins and that size of side panels can't be currently controlled.

Reply:
I also added anchor aliases, e.g. `center_top` is just `top`, so that `location` in side panels can be specified more sensibly.